docs/reference/troubleshoot.md

```diff
@@ -7,12 +7,22 @@ indent: true
 
 # Troubleshooting
 
+* [Requested Resource Not Found]
 * [Resource Status and Conditions]
 * [Resource Events]
 * [Crossplane Logs]
 * [Pausing Crossplane]
 * [Deleting a Resource Hangs]
 
+## Requested Resource Not Found
+
+If you use kubectl Crossplane plugin to install a `Provider` or `Configuration` -
+e.g. `kubectl crossplane install provider crossplane/provider-aws:master` and
```

Comment:
Could you put in parens instead of dash here or add a concluding dash after the interjected statement?